### ISSUE TITLE
Hosted version of the console

### ISSUE THREAD

**Issue by sampoder** [2bttns/2bttns#55]
Are there plans to offer a hosted version of the 2bttns console?

**Comment by mriziq** [comment]
Yes, that's the next big move. We're just starting with that, so if you want to talk more about it, feel free to email me amer@2bttns.com OR talk on Discord [2bttns discord](https://discord.com/invite/Karu5RrpdU)

This makes me think we should publish our Roadmap, too.
